src/Citeproc/Types.hs

```diff
@@ -1438,6 +1448,7 @@ removeEmptyStrings = filter (not . isEmptyString)
 
 data Output a =
     Formatted Formatting [Output a]
+  | Linked LinkType Text [Output a]
```

Comment:
Keeping track of the `LinkType` as part of the output was necessary so that the appearance of links can be controlled by flags in `CiteprocOptions`.

Reply:
Why do we need to distinguish these?
I would have thought that any Linked would simply be rendered as a hyperlink to the given URL (Text) with the `[Output a]` as anchor.

Reply:
Oh, I see -- the point is not to affect *how* it is rendered, but whether the link is rendered at all.
Couldn't this be done earlier, though?  Don't insert the Linked constructor in the first place unless the options call for that sort of link.

Reply:
Yeah, technically the decision whether to create a link or not could be moved to [`Eval.hs/evalItem`](https://github.com/jgm/citeproc/blob/master/src/Citeproc/Eval.hs#L1128) where the `Linked` constructor is actually used.  However, we'd need to pass the `CiteprocOptions` along into the evaluation [`Context`](https://github.com/jgm/citeproc/blob/master/src/Citeproc/Eval.hs#L41), since they aren't currently available inside the eval functions.  Is that fine with you?

Reply:
Ah, I see.  Eval has no access to CiteprocOptions.  Well, I think the way you're doing it is okay, then.